internal/compiler-bridge/src-2.10/main/scala/xsbt/API.scala

```diff
@@ -1,81 +0,0 @@
-/* sbt -- Simple Build Tool
```

Comment:
Because there's no sbt/util for 2.10 https://github.com/sbt/util/pull/44/commits/4e4aa08a1a13bd85c70d60d7129438acd1819406